tasks/create_packages.js

```diff
@@ -36,28 +36,6 @@ function createPackages(grunt) {
       // kibana.tar.gz
       commands.push([ 'tar', '-zcf', archiveName + '.tar.gz', name ]);
 
```

Comment:
I think you can pass a parameter to tell it to only build for linux, right? this may be unnecessary